client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+				),
+				planOne: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planTwo: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+			},
+			{
+				featureName: translate( 'eCommerce' ),
+				tooltip: translate(
+					'Build, manage, and scale your online store with unlimited products or services, multiple currencies, integrations with top shipping carriers, and eCommerce marketing tools.'
+				),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+			},
+			{
+				featureName: translate( 'Custom domain name' ),
+				tooltip: translate(
+					'Remove “.wordpress.com” from your site address with a custom domain of your choosing. Free for one year (will renew at its regular price).'
+				),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: {
+					included: false,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: 'Free domain for the first year.',
+				},
+			},
+			{
+				featureName: translate( 'SEO' ),
+				tooltip: translate(
+					'SEO (Search Engine Optimization) helps your site rank higher, and more accurately in search engines, so more people can find your site.'
+				),
+				planOne: {
+					included: false,
+					copy: '-',
+					mobileCopy: null,
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Simple' ),
+					mobileCopy: translate( 'Simple SEO' ),
+				},
+			},
+			{
+				featureName: translate( 'Customer support' ),
+				tooltip: translate(
+					'Keep moving forward with your site at any time. Our Happiness Engineers will help you via email, live chat, or 24/7 priority live chat, depending on the plan you choose.'
+				),
+				planOne: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Email support' ),
+					mobileCopy: translate( 'Email support' ),
+				},
+			},
+			{
+				featureName: translate( 'Remove WordPress.com Ads' ),
+				tooltip: translate(
+					'Allow your visitors to visit and read your website without seeing any WordPress.com advertising.'
+				),
+				planOne: { included: false, copy: translate( 'No' ), mobileCopy: null },
+				planTwo: { included: true, copy: translate( 'Yes' ), mobileCopy: null },
+			},
+		],
+	};
+
+	const sharedFeatures = {
+		Limited: [
+			{
+				icon: pages,
+				description: translate( 'Unlimited posts and pages' ),
+			},
+			{
+				icon: lifesaver,
+				description: translate( 'Best-in-class managed hosting' ),
+			},
+			{
+				icon: lifesaver,
+				description: translate( 'Free SSL certificante' ),
+			},
+			{
+				icon: lifesaver,
+				description: translate( '6GB of storage' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( '6 GB of storage space.' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tabbed-plan-signup-test)